components/builder-scheduler/src/config.rs

```diff
@@ -71,7 +71,7 @@ impl Default for Config {
         Config {
             routers: vec![SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::new(127, 0, 0, 1), 5562))],
             job_servers: vec![jobsrv],
-            datastore_connection_url: String::from("postgresql:://hab@127.0.0.1/builder_db_test"),
```

Comment:
ha good catch--I'm going to guess the double colon was some Rustacean muscle memory